Add sensor user journey (#2986)
viamrobotics/docs: static/include/how-to/gather-sync-sensor.md

Enable **Syncing** to ensure captured data is synced to the cloud and set the sync interval, for example to `0.05` minutes to sync every 3 seconds.

{{< /tablestep >}}
{{< tablestep >}}
{{<imgproc src="/icons/components/sensor.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="configure a camera component">}}
**2. Capture data from sensor**

On the **CONFIGURE** tab, go to the **sensor**'s card and find the **Data capture** section.
Add a new method, `Readings`, to capture data for and set the frequency.
For example, setting a frequency of `0.1` will capture data once every ten seconds.

{{< /tablestep >}}
{{< tablestep >}}
{{<imgproc src="/services/ml/configure.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
**3. Save to start capturing**

Save the config.
With cloud sync enabled, captured data is automatically uploaded to the Viam app after a short delay.

{{< /tablestep >}}
{{< tablestep link="/services/data/view/">}}
{{<imgproc src="/services/icons/data-capture.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Capture tabular data from a sensor">}}

> **4. View data**

Author: ```suggestion
**4. View data in the Viam app**
```

Viam's [data management service](/services/data/) lets you capture data locally from sensors and then sync it to the cloud where you can access all data across different {{< glossary_tooltip term_id="machine" text="machines" >}} or {{< glossary_tooltip term_id="location" text="locations" >}} and see trends over time.

{{< table >}}
{{< tablestep link="/services/data/">}}
{{<imgproc src="/services/icons/data-management.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Configure the data management service">}}
**1. Add the data management service**

> On your machine's **CONFIGURE** tab, add the **data management** service.

Reviewer: maybe more steps here like with the sensors?

Author: done

Enable **Syncing** to ensure captured data is synced to the cloud and set the sync interval, for example to `0.05` minutes to sync every 3 seconds.

{{< /tablestep >}}
{{< tablestep >}}
{{<imgproc src="/services/icons/data-management.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Configure the data management service">}}
**2. Tag your data**

Add a tag to your machine's captured data, for example `machine1-sensor-data`. This makes querying data easier because you can query by tag.

{{< /tablestep >}}
{{< tablestep >}}
{{<imgproc src="/icons/components/sensor.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="configure a camera component">}}
**3. Capture data from sensor**

> On the **CONFIGURE** tab, go to the **sensor's** card and find the **Data capture** section. Add a new method, `Readings`, to capture data for and set the frequency, for example to `0.1` readings per second. This will capture data once every ten seconds.

Reviewer: ```suggestion
On the **CONFIGURE** tab, go to the **sensor**'s card and find the **Data capture** section.
Add a new method, `Readings`, to capture data for and set the frequency, for example to `0.1` readings per second. This will capture data once every ten seconds.
```

{{< /tablestep >}}
{{< tablestep >}}
{{<imgproc src="/services/ml/configure.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
**4. Save to start capturing**

> Save the config. With cloud sync enabled, captured data is automatically uploaded to the Viam app after a short delay.

Reviewer: ```suggestion
Save the config.
With cloud sync enabled, captured data is automatically uploaded to the Viam app after a short delay.
```

{{< /tablestep >}}
{{< tablestep link="/services/data/view/">}}
{{<imgproc src="/services/icons/data-capture.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Capture tabular data from a sensor">}}
**5. View data**

> To confirm data is being synced, go to the **DATA** tab and select the **Sensors** subtab. Confirm that you are seeing data appear.

Reviewer: ```suggestion
To confirm data is being synced, go to the **DATA** tab and select the **Sensors** subtab.
Confirm that you are seeing data appear.
```

> Viam's [data management service](/services/data/) lets you capture data locally from sensors and then sync it to the cloud where you can access all data across different {{< glossary_tooltip term_id="machine" text="machines" >}} or {{< glossary_tooltip term_id="location" text="locations" >}} and see trends over time.

Reviewer 2: "and see trends over time" feels like a bit much since the app itself doesn't have built-in data visualization tools and it'd be somewhat hard to see trends over time just by looking at raw data. I think it would make it easier to read if this sentence was a little shorter anyway:
```suggestion
Viam's [data management service](/services/data/) lets you capture data locally from sensors and then sync it to the cloud where you can access all data across different {{< glossary_tooltip term_id="machine" text="machines" >}} or {{< glossary_tooltip term_id="location" text="locations" >}}.
```

{{< table >}}
{{< tablestep link="/services/data/">}}
{{<imgproc src="/services/icons/data-management.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Configure the data management service">}}
**1. Add the data management service**

On your machine's **CONFIGURE** tab, add the **data management** service.

Enable **Syncing** to ensure captured data is synced to the cloud and set the sync interval, for example to `0.05` minutes to sync every 3 seconds.

{{< /tablestep >}}
{{< tablestep >}}
{{<imgproc src="/services/icons/data-management.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Configure the data management service">}}
**2. Tag your data**

Add a tag to your machine's captured data, for example `machine1-sensor-data`. This makes querying data easier because you can query by tag.

{{< /tablestep >}}
{{< tablestep >}}
{{<imgproc src="/icons/components/sensor.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="configure a camera component">}}
**3. Capture data from sensor**

On the **CONFIGURE** tab, go to the **sensor**'s card and find the **Data capture** section.

> Add a new method, `Readings`, to capture data for and set the frequency, for example to `0.1` readings per second. This will capture data once every ten seconds.

Reviewer 2: Looks like there's an extra "for"
```suggestion
Add a new method, `Readings`, to capture sensor readings and set the frequency, for example to `0.1` readings per second.
This will capture data once every ten seconds.
```
Seems like this could be less hand-holdy or at least more succinct since the app says what the units are.
Briefest:
```suggestion
Add a new method, `Readings`, and set the frequency at which to capture sensor readings.
```
Medium-brief:
```suggestion
Add a new method, `Readings`, to capture data for and set the frequency.
For example, setting a frequency of `0.1` will capture data once every ten seconds.
```
